openapi/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.8"
+
+services:
+  swagger-cli:
```

Comment:
Why is this necessary?

Reply:
this part is for running a container that will create the combined openapi file